docs/can-fixture.md

```diff
@@ -9,145 +9,225 @@
 
 @signature `fixture(ajaxSettings, requestHandler(...))`
 
-If an XHR request matches ajaxSettings, calls requestHandler with the XHR requests data. Makes the XHR request respond with the return value of requestHandler or the result of calling its response argument.
+  If an XHR request matches ajaxSettings, calls requestHandler with the XHR requests data. Makes the XHR request respond with the return value of requestHandler or the result of calling its response argument.
 
-The following traps requests to GET /todos and responds with an array of data:
+  The following traps requests to GET /todos and responds with an array of data:
 
-```js
-fixture( { method: "get", url: "/todos" },
-	function( request, response, headers, ajaxSettings ) {
-		return {
-			data: [
-				{ id: 1, name: "dishes" },
-				{ id: 2, name: "mow" }
-			]
-		};
-	} );
-```
+  ```js
+  import {fixture} from "can";
+  import "//unpkg.com/jquery@3.3.1/dist/jquery.js";
 
-When adding a fixture, it will remove any identical fixtures from the list of fixtures. The last fixture added will be the first matched.
+  fixture( { method: "get", url: "/todos" },
+    ( request, response, headers, ajaxSettings ) => {
+      return {data: [
+        { id: 1, name: "dishes" },
+        { id: 2, name: "mow" }
+      ]};
+    }
+  );
 
-  @param {can-fixture/types/ajaxSettings} ajaxSettings An object that is used to match values on an XHR object, namely the url and method. url can be templated like /todos/{_id}.
+  $.get("/todos").then(result => {
```

Comment:
I would probably show either using `can-ajax` or the native `XMLHTTPRequest` object.

Reply:
Resolved in c5685a9 and 574c32a (might be missing some in can-fixture.md).